experiment/src/org/labkey/experiment/api/ExperimentServiceImpl.java

```diff
@@ -3927,7 +3928,8 @@ public void deleteProtocolByRowIds(Container c, User user, int... selectedProtoc
             transaction.commit();
         }
 
-        AssayService.get().unindexAssays(Collections.unmodifiableCollection(expProtocols));
+        if (null != assayService)
```

Comment:
```suggestion
        if (assayService != null)
```
for consistency with other calls.